### PR TITLE
Make audio latency configurable

### PR DESCRIPTION
**Pull request by ArniDagur**
Result of discussion from https://github.com/ppy/osu-framework/pull/3832

### CODE REVIEW

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Framework/Audio/AudioManager.cs [hunk 1]
```diff
@@ -59,6 +59,14 @@ public class AudioManager : AudioCollectionManager<AdjustableAudioComponent>
         /// </summary>
         public readonly Bindable<string> AudioDevice = new Bindable<string>();
 
+        /// <summary>
+        /// TODO: Documentation
+        /// </summary>
+        public readonly BindableInt DevicePeriod = new BindableInt(5) {
+            MinValue = 1,
+            MaxValue = 240
```
[the remark this answers is not in the document]
Comment (ArniDagur) [reply osu.Framework/Audio/AudioManager.cs:68]: Probably not. Anything above 100 or so for me is unplayable. I think I'll change the max to 60 or so later.

**Comment by bdach** [comment]
@ArniDagur please avoid force-pushing in general if you can avoid it.

File: osu.Framework/Audio/AudioManager.cs [hunk 2]
```diff
@@ -59,6 +59,51 @@ public class AudioManager : AudioCollectionManager<AdjustableAudioComponent>
...
+        /// <remarks>
+        /// If this setting is changed at runtime, it will only apply to newly
+        /// created BASS channels.
+        /// </remarks>
+        public readonly BindableInt PlaybackBufferSize = new BindableInt(100)
+        {
+            // BASS default is 500 ms.
+            MinValue = 10,
+            MaxValue = 5000
+        };
```
Comment (peppy) [inline osu.Framework/Audio/AudioManager.cs:105]: shouldn't be added

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by smoogipoo** [comment]
Closing due to inactivity. Still need to discuss how to do this since we all have different ideas/expectations.

**Pull request closed without merging**